Sources/Orbit/Components/Alert.swift

```diff
@@ -30,33 +30,44 @@ public struct Alert<Content: View>: View {
 
     @Environment(\.idealSize) var idealSize
 
+    let style: AlertStyle
     let title: String
     let description: String
     let iconContent: Icon.Content
     let buttons: AlertButtons
     let status: Status
     let isSuppressed: Bool
     let descriptionLinkAction: TextLink.Action
-    @ViewBuilder let content: Content
+    @ViewBuilder let customContent: Content
 
     public var body: some View {
-        HStack(alignment: .top, spacing: .xSmall) {
-            Icon(content: iconContent)
-                .foregroundColor(status.color)
-                .accessibility(.alertIcon)
-            
+        content
+            .padding(.leading, iconContent.isEmpty ? .medium : .small)
+            .background(background)
+            .cornerRadius(BorderRadius.default)
+            .accessibilityElement(children: .contain)
+    }
+
+    @ViewBuilder var content: some View {
+        switch style {
+            case .default:  defaultContent
+            case .inline:   inlineContent
+        }
+    }
+
+    @ViewBuilder var defaultContent: some View {
+        HStack(alignment: .firstTextBaseline, spacing: .xSmall) {
```

Comment:
For both `defaultContent` and `inlineContent`, the `.top` alignment is more precise and preferred for constructing Orbit components (so it matches design heights 1:1). 

The `.firstTextBaseline` alignment for Icon+Text is approximate and is meant for use where Icon and Text sizes (Icon size and text line height to be exact) do not match.

Reply:
`.top` really doesn't look good for single-line content though.

With `.firstTextBaseline`:
<img width="373" alt="Screenshot 2023-01-07 at 9 17 27" src="https://user-images.githubusercontent.com/12349477/211141075-e99ed48e-f187-47f2-9d3c-f814c3c2e515.png">

With `.top`
<img width="376" alt="Screenshot 2023-01-07 at 9 17 15" src="https://user-images.githubusercontent.com/12349477/211141086-fe8f6930-fd2e-41ce-899f-e96e98072919.png">

Reply:
True. Designs use `centered` as they use single-line example, which is not useful. But they visually seem to match the baseline alignment 👍 

The leading icon + text should still be stacked together using `top` alignment, with the inline button being aligned to `firstbaseline` in outer HStack.

Reply:
Ok, two stacks it is.